spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/SparkTestBase.java

```diff
@@ -39,6 +39,7 @@
 import org.junit.AfterClass;
 import org.junit.Assert;
 import org.junit.BeforeClass;
+import org.junit.internal.ExactComparisonCriteria;
```

Comment:
Is it normal to use this class that's got `internal` in the package name? Do we have to worry about behavioral changes if we upgrade JUnit?

Reply:
Thanks for your review, I have modified the code